StyleCop.Analyzers/StyleCop.Analyzers/ReadabilityRules/SA1130UseLambdaSyntax.cs

```diff
@@ -117,7 +117,7 @@ private static bool HandleMethodInvocation(SemanticModel semanticModel, Anonymou
                 var invocationExpression = originalInvocationExpression.ReplaceNode(anonymousMethod, lambdaExpression);
                 SymbolInfo newSymbolInfo = semanticModel.GetSpeculativeSymbolInfo(location.SourceSpan.Start, invocationExpression, SpeculativeBindingOption.BindAsExpression);
 
-                if (originalSymbolInfo.Symbol != newSymbolInfo.Symbol)
+                if (!originalSymbolInfo.Symbol.Equals(newSymbolInfo.Symbol))
```

Comment:
❔ Are we sure `originalSymbolInfo.Symbol` cannot be null?

Reply:
Yes, or else `GetDelegateParameterList` would have thrown NRE due to the call on line 107. If that's too distant of a guarantee, I have no problem using `Equals`.